### DISCUSSION TITLE
How to overwrite a certain ANSI Color

### DISCUSSION THREAD

**Discussion by bn-rstaudacher** [wezterm/wezterm#6311] [Q&A]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

kwin

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

Hi,

I have the problem some console text is hard to read with the theme I am using. Because of this I want to change the color of a certain ANSI color ('\x1b[90m'). What is the best way to do that?

Thanks

**Comment by bew** [comment]
Did you have a look at https://wezfurlong.org/wezterm/config/appearance.html#defining-your-own-colors ?
